README.md

```diff
@@ -1,37 +1,40 @@
-| ---------------- | ---------------------------------------------------------------------------------------------------- |
-| Alpine           | `3.9.4` `Alpine` `Alpine-3.9.4` `amd64` `amd64-Alpine` `amd64-Alpine-3.9.4`                          |
-| Amazon Linux     | `2018.03` `AmazonAMI` `AmazonAMI-2018.03` `amd64` `amd64-AmazonAMI` `amd64-AmazonAMI-2018.03`        |
-| Amazon Linux 2   | `2` `Amazon` `Amazon-2` `amd64` `amd64-Amazon` `amd64-Amazon-2`                                      |
-| CentOS 6         | `6.10` `CentOS` `CentOS-6.10` `amd64` `amd64-CentOS` `amd64-CentOS-6.10`                             |
-| CentOS 7         | `7.7.1908` `CentOS` `CentOS-7.7.1908` `amd64` `amd64-CentOS` `amd64-CentOS-7.7.1908`                 |
-| Debian 9         | `9.11` `Debian` `Debian-9.11` `amd64` `amd64-Debian` `amd64-Debian-9.11`                             |
-| Debian 10        | `10` `Debian` `Debian-10` `amd64` `amd64-Debian` `amd64-Debian-10`                                   |
-| Debian testing   | `testing` `Debian` `Debian-testing` `amd64` `amd64-Debian` `amd64-Debian-testing`                    |
-| FreeBSD 11       | `11.1-STABLE` `amd64` `amd64-freebsd` `amd64-freebsd-11.1-STABLE` `freebsd` `freebsd-11.1-STABLE`    |
-| FreeBSD 12       | `12.0-RELEASE` `amd64` `amd64-freebsd` `amd64-freebsd-12.0-RELEASE` `freebsd` `freebsd-12.0-RELEASE` |
-| Oracle Linux 6   | `6.10` `OracleServer` `OracleServer-6.10` `amd64` `amd64-OracleServer` `amd64-OracleServer-6.10`     |
-| Oracle Linux 7   | `7.6` `OracleServer` `OracleServer-7.6` `amd64` `amd64-OracleServer` `amd64-OracleServer-7.6`        |
-| Oracle Linux 8   | `8.0` `OracleServer` `OracleServer-8.0` `amd64` `amd64-OracleServer` `amd64-OracleServer-8.0`        |
-| Ubuntu 14.04     | `14.04` `Ubuntu` `Ubuntu-14.04` `amd64` `amd64-Ubuntu` `amd64-Ubuntu-14.04`                          |
-| Ubuntu 16.04     | `16.04` `Ubuntu` `Ubuntu-16.04` `amd64` `amd64-Ubuntu` `amd64-Ubuntu-16.04`                          |
-| Ubuntu 18.04     | `18.04` `Ubuntu` `Ubuntu-18.04` `amd64` `amd64-Ubuntu` `amd64-Ubuntu-18.04`                          |
-| Windows 10       | `10.0` `amd64` `amd64-windows` `amd64-windows-10.0` `windows` `windows-10.0`                         |
-
-On Linux computers, the plugin uses the output of the `[lsb_release](https://linux.die.net/man/1/lsb_release)` command.
-If `lsb_release` is not installed, labels on Linux agents will be guessed based on values in /etc/os-release.
-When /etc/os-release is used, less detailed labels are provided.
-For example:
-
-| Platform         | Labels assigned by plugin when lsb-release is not installed     |
-| ---------------- | --------------------------------------------------------------- |
-| CentOS 7         | `7` `CentOS` `CentOS-7` `amd64` `amd64-CentOS` `amd64-CentOS-7` |
-
-Source code is formatted with the command:
-
-    mvn com.coveo:fmt-maven-plugin:format
+# Platform Labeler Plugin

```

Comment:
It looks like this changed the line ending characters in the file.  I strongly prefer that we avoid large scale whitespace changes like this change of line ending characters.  When I compare your change with white space differences ignored, I like the change.

Could you reduce the differences by retaining the same line endings as before?

Reply:
Oh I did not intend to change the line endings, I'll revert them and/or align with existing line-ending